### ISSUE TITLE
Legacy release download address for MacOS

### ISSUE THREAD

**Issue by BerginJay** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#879]
## Description

Open the following URL, and click `All RELEASE` botton. 

Page URL: https://qgis.org/en/site/forusers/download.html

The download URL for OS X in `Previous releases of QGIS are still available here - including older releases for OS X here` is:

https://qgis.org/downloads/macOS/

It contain QIGS version from 2.14 to 3.8. However, https://qgis.org/downloads/macos/ contains more recent versions of QGIS.

So I suggest that we should merge these two or change link address in `Previous releases of QGIS are still available here - including older releases for OS X here`.

**Comment by DelazJ** [comment]
See also #463 for Windows releases

**Comment by BerginJay** [comment]
So is this a feature?

**Comment by kyngchaos** [comment]
I think what happened here is that, after a long time of releasing the Mac package on my website, I made it available directly at QGIS.org in the macOS folder (that's why there are only a couple years of releases there).  Then I passed the "official" build to Petric as an all-in-one package and he created a new folder with all lower case, macos.

So, the dual folder setup just happened.  It may be a good idea to move the older packages into the current folder as a subfolder.  I don't think there any internal links that will break, but there may be external links to these old packages.  I do keep a full archive of my old packages on my site.
